src/matter/cluster/Cluster.ts

```diff
@@ -28,16 +28,16 @@ export const enum AccessLevel {
 }
 
 /* Interfaces and helper methods to define a cluster attribute */
-export interface Attribute<T> { id: number, schema: TlvSchema<T>, optional: boolean, readAcl: AccessLevel, writable: boolean, writeAcl?: AccessLevel, default?: T }
+export interface Attribute<T> { id: number, schema: TlvSchema<T>, optional: boolean, readAcl: AccessLevel, writable: boolean, nonVolatile?: boolean, omitChanges?: boolean, writeAcl?: AccessLevel, default?: T }
```

Comment:
nonVolatile => persistent?
Better to always use positive names in flag, otherwise this adds cognitive burden, like this:
`if (!nonVolatile) return;`

:-)

Same for omitChanges => observable (or something like this)

I don't think nonVolatile / omitChanges should be optional: a default value can always defined in the declaration methods below.
writeAcl is optional because it only makes sense when the attribute is writable
default is optional since not all the attributes have a default

Reply:
Naming wise we are then (again) on the question how close to be With the specs when it comes to names ;-) it is officially called that way there.

But yes I will rename it then.

Reply:
Ok for nonVolative -> persistent ... all fine.

But for omitChanges aka "do NOT send updates on subscriptions" i can not find a positive name that would not also "reverse" the meaning ... And if we go to "subscribable" as the "positive opposite" then User (and we) would need to have a big brain effort to map with the specs because this is then not "clear" (because if the specs say "omit changes" we would need to set "subscribable=false" with a default of true. Additionally according to specsit is indeed "subscribable" (aka would not rturn an error on subscribe) but would in fact never get any value, so also the name would be misleading.